src/components/TestimonialSubmit/index.jsx

```diff
@@ -144,7 +144,8 @@ const Testimonial = () => {
                 <div className="container m-0 p-2 pb-5">
                     <div className="row justify-content-left pb-3">
                         <p className={`${styles.caption}`}>
-                            Description here
+                            If you are a University of Ottawa student who has signed the petition, we would love to hear your thoughts! Comments submitted in this form will be subject to review by the uPetition team and, if they are accepted, will be displayed on our Testimonials page.<br /><br />
+                            Note: Our Testimonials page is currently under construction, but the team is collecting student thoughts while we build!
```

Comment:
I don't know what you're using for i18n but I would assume this should be in a translations file somewhere.

Reply:
to be honest I don't know if i18n is really possible right now since I don't think anyone is willing to translate for free at the rate I'm pushing out code, and we don't yet have the implementation of the multilingual support after the initial translations that were done. There's a backlog ticket to do it at the moment